test/test_optimization.py

```diff
@@ -409,7 +407,6 @@ def test_solver_options(model, solver, io_api):
         "gurobi": {"TimeLimit": 1},
         "glpk": {"tmlim": 1},
         "cplex": {"timelimit": 1},
-        "scip": {"time_limit": 1},
```

Comment:
I have no idea, if this is correct. But the second "scip" overrode the first one, anyway. So the behaviour should be the same.

Reply:
The second one is also the newer commit, so it's good